package.json

```diff
@@ -17,7 +17,8 @@
     "yargs": "12.0.5",
     "webpack": "^4.0.0",
     "html-webpack-plugin": "^3.2.0",
-    "axios": "0.18.0"
```

Comment:
Why axios is removed ? It's not used ?

Reply:
The line `"axios": "0.18.0"` was removed. And the new line `"axios": "0.18.0",` (with `,`) was added.